drizzlepac/hapsequencer.py

```diff
@@ -23,8 +24,8 @@
 from stwcs import wcsutil
 
 __taskname__ = 'hapsequencer'
-log_level = logging.INFO
-log = logutil.create_logger(__name__, level=log_level, stream=sys.stdout)
+log_level = logging.ERROR
```

Comment:
I am not sure what this assignment is actually doing.

Reply:
I think you are looking at an older version. These lines are different in the latest version.